kore/src/Kore/Profiler/Data.hs

```diff
@@ -118,12 +119,12 @@ profileDurationEvent tags action = do
 
 {- Times an action in the format required by @ghc-events-analyze@.
 -}
-profileDurationStartEnd
+profileDurationStartStop
     :: MonadIO profiler => [String] -> profiler a -> profiler a
-profileDurationStartEnd event action = do
-    liftIO $ traceEventIO ("START " ++ show event)
+profileDurationStartStop event action = do
+    liftIO $ traceEventIO ("START " ++ List.intercalate "/" event)
     a <- action
-    liftIO $ traceEventIO ("END " ++ show event)
+    liftIO $ traceEventIO ("STOP " ++ List.intercalate "/" event)
```

Comment:
https://github.com/kframework/kore/blob/master/kore/app/profiler/Main.hs parses this output, so you should probably update it (unless there's some magic ).

Reply:
Sorry, I accidentally pressed "Comment" too soon. The link should be
https://github.com/kframework/kore/blob/master/kore/app/profiler/Main.hs#L249
and the text in the parethesis should end with "magic that makes `readMaybe` resilient to these changes."